dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js

```diff
@@ -110,21 +109,9 @@ const findOneGefDeal = async (_id, callback) => {
   const deal = await dealsCollection.findOne({ _id: ObjectID(_id) });
 
   if (deal) {
-    const promises = await Promise.all([
-      findAllGefFacilitiesByDealId(_id),
-      findOneUser(deal.userId),
-    ]);
-
-    const [
-      facilities,
-      maker,
-    ] = [...promises];
+    const facilities = await findAllGefFacilitiesByDealId(_id);
 
     deal.facilities = facilities;
-
-    if (maker) {
```

Comment:
Should this be removed?

Reply:
Yes, at this point, with these changes - the deal will already have the maker, so no need to get user from DB collection and manually add to the deal here like it was before 😄